packages/amplify-category-storage/src/provider-utils/awscloudformation/cdk-stack-builder/ddb-stack-builder.ts

```diff
@@ -21,60 +24,55 @@ export class AmplifyDDBResourceStack extends cdk.Stack implements AmplifyDDBReso
   }
 
   /**
-   *
-   * @param props :cdk.CfnOutputProps
-   * @param logicalId: : lodicalId of the Resource
+   * adds  cfn output to stack
    */
   addCfnOutput(props: cdk.CfnOutputProps, logicalId: string): void {
     try {
+      // eslint-disable-next-line no-new
       new cdk.CfnOutput(this, logicalId, props);
```

Comment:
out of curiosity. I noticed this pattern of few places. I.e. we `new` something out of the blue and we drop reference. Does ctor attach it somehow to `this` ?

Reply:
yeah , ctor attaches this output to stack and return the value